Docs: fix id token hint information (#79890)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md

As an example, `<PROVIDER_DOMAIN>` can be `keycloak-demo.grafana.org`,
`<REALM_NAME>` can be `grafana` and `<GRAFANA_DOMAIN>` can be `play.grafana.org`.

> > **Note**: Grafana supports ID token hint for single logout. If OAuth is detected as the authentication method, Grafana will automatically add the `id_token_hint` parameter to the logout request.

Reviewer: ```suggestion
{{% admonition type="note" %}}
Grafana supports ID token hints for single logout. Grafana automatically adds the `id_token_hint` parameter to the logout request if it detects OAuth as the authentication method.
{{% /admonition %}}
```
